adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -1138,74 +1138,110 @@ func (swagger *MgwSwagger) PopulateSwaggerFromAPIYaml(apiData APIYaml, apiType s
 	if endpointConfig.ImplementationStatus == prototypedAPI {
 		swagger.IsProtoTyped = true
 	}
-	if len(endpointConfig.SandBoxEndpoints) > 0 {
+
+	if len(endpointConfig.ProductionEndpoints) > 0 {
 		var endpoints []Endpoint
 		endpointType := LoadBalance
-		for _, endpointConfig := range endpointConfig.SandBoxEndpoints {
-			sandBoxEndpoint, err := getHostandBasepathandPortWebSocket(endpointConfig.Endpoint)
-			if err == nil {
-				endpoints = append(endpoints, *sandBoxEndpoint)
+		var unProcessedURLs []interface{}
+		for _, endpointConfig := range endpointConfig.ProductionEndpoints {
+			if apiType == WS {
+				prodEndpoint, err := getHostandBasepathandPortWebSocket(endpointConfig.Endpoint)
```

Comment:
Shall we rename this function?
```suggestion
				prodEndpoint, err := getEndpointForWebsocketURL(endpointConfig.Endpoint)
```

Reply:
added https://github.com/wso2/product-microgateway/pull/2545/commits/038c7debac61e02744c4247a9e44e33bd4fe8a1b